src/pages/Forgot-password/__test__/ForgorPassword.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render, screen } from '@testing-library/react';
+import ForgotPassword from "../forgot-password";
+
+test('renders some text', () => {
+    render(<ForgotPassword />);
+    const linkElement = screen.getByText(/Slightly Techie/i);
+    expect(linkElement).toBeInTheDocument();
+    
```

Comment:
This is a bad test. Why not check for something related to "forgot password" ??

Reply:
Okay Sir